### DISCUSSION TITLE
request: toggle multiple audio tracks off and on

### DISCUSSION THREAD

**Discussion by chocolate-elvis** [videovillage/community#114] [Ideas]
Thanks for taking this on. We need a real post production capable player for modern MacOS...

While often delivering 8 channels of audio (a 5.1 and a stereo) in a single file to clients, would be great for QC purposes to be able to turn off channels. Thanks!

**Comment by gregcotten** [comment]
Hi! Yes, we definitely need more granular audio controls for Screen. In your example, do you have 8 audio tracks, or really just 2 audio tracks with multiple channels in each (one tagged stereo, the other tagged 5.1)

**Comment by Kingofmyroom** [comment]
I would also love to see this! My camera shoots 4 channels of audio, two of those being scratch tracks. It would be great to be able to quickly mute those tracks and just hear the audio from other tracks.

Thanks!

**Comment by jtb-git** [comment]
Yes please! Just the option to mute just channels 1/2 for example would be a huge timesaver!!

**Comment by Darmouthz** [comment]
+1. I really feel toggling between audio channels, at least for monitoring, is very basic. We've had this in VLC for years. At that price point, I was pretty disappointed to find out I can't do it. It makes all my multichannel recordings unwatchable with sound.
